torch_xla/csrc/pooling.h

```diff
@@ -59,7 +67,8 @@ xla::XlaOp BuildAdaptiveAvgPool2dBackward(xla::XlaOp out_backprop,
 
 // Returns true if XLA lowering is supported for the given input and output size
 // combination.
-bool IsSupportedAdaptiveAvgPool2d(absl::Span<const xla::int64> input_size,
-                                  absl::Span<const xla::int64> output_size);
+bool IsSupportedAdaptiveAvgPool(absl::Span<const xla::int64> input_size,
+                                absl::Span<const xla::int64> output_size,
+                                int unsigned pool_dim);
```

Comment:
Pool_dim refers to how number of dimensions for requested pooling. 2 for 2d, 3 for 3d, I dont see any scenario when it would be negative? Is there a potential scenario for which we should choose int instead of int unsigned?

Reply:
We do not use unsigned int almost anywhere in existing code.
Do not be the black fly over the white sheet 😉

Reply:
Please do not mark `resolved` if the code does not fix it.

Reply:
`int unsigned` --> `int`

Reply:
fixed.